internal/pkg/service/buffer/storage/statistics/repository/watch.go

```diff
@@ -0,0 +1,12 @@
+package repository
+
+import (
+	"context"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/statistics"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/common/etcdop"
+)
+
+func (r *Repository) GetAllAndWatch(ctx context.Context) *etcdop.RestartableWatchStreamT[statistics.Value] {
+	return r.Schema().GetAllAndWatch(ctx, r.client)
+}
```

Comment:
Pridal som metodu `statistics.Repository.GetAllAndWatch`, ktoru pouziva `etcdop.Mirror` v `cache.L1`.

Povodne sa to riesilo az v cache, ale takto tam nemusi ist `etcd client + serde`, staci tam dat ako zavislost iba `Repository`.